profiles/package.mask

```diff
@@ -508,9 +508,9 @@ net-misc/wicd
 
 # Andrew Ammerlaan <andrewammerlaan@riseup.net> (2020-01-26)
 # Proxy Maintainers <proxy-maint@gentoo.org>
-# v3.6.3 uses huge amounts of memory: Bug #705682
+# v3.6.4 uses huge amounts of memory: Bug #705682
 # v3.6.1 is the latest version without this issue
-=dev-libs/libsass-3.6.3
+=dev-libs/libsass-3.6.4
```

Comment:
We don't know how long this is going to go on for and we want to keep 3.6.3 masked. I would have expected 3.6.4 to fix it but here we are so best make it `>=dev-libs/libsass-3.6.2`.

Reply:
Good idea,

I thought I read somewhere that the next release will be 4.0 so hopefully it will be fixed in that release.